From: Stefano Rivera <stefanor@debian.org>
Date: Sat, 30 Jan 2021 19:38:46 -0700
Subject: Disable pytest-cov

We don't need coverage reports for our use-case. And this avoids us
needing to provide setup.cfg for the test run.

Forwarded: not-needed
---
 setup.cfg | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index fa392d1..6505000 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,5 +1,3 @@
-[coverage:run]
-
 [flake8]
 exclude = *.egg*,.env,.git,.tox,_*,build*,dist*,venv*,python2/,python3/
 ignore = E261,E731,W503
@@ -8,8 +6,6 @@ max-line-length = 121
 [tool:pytest]
 minversion = 3.2
 addopts = 
-	--cov-config=setup.cfg
-	--cov=httplib2
 	--ignore=tests/_TODO_legacy/
 	--noconftest
 	--showlocals
